saleor/graphql/app/mutations.py

```diff
@@ -382,7 +393,7 @@ def clean_manifest_url(self, url):
             url_validator(url)
         except (ValidationError, AttributeError):
             msg = "Enter a valid URL."
-            code = AppErrorCode.INVALID.value
+            code = AppErrorCode.INVALID_MANIFEST_URL.value
```

Comment:
```
{ field: "manifest_url", code="INVALID_MANIFEST_URL" }
```
Don't provide more information than
```
{ field: "manifest_url", code="INVALID" }
```
Maybe we should use `INVALID_URL_FORMAT" or just stay with the previous version?